vignettes/plotting_theme.Rmd

```diff
@@ -0,0 +1,309 @@
+
+For example, this is a selection of 7 colors from the **"Viridis"** palette:  
+
+```{r include = FALSE}
+fig <- ggplot(data.frame(x = as.factor(seq(7, 1))), aes(x = 1, y = 1, fill = x)) +
+  geom_col(position = position_stack(reverse = TRUE), show.legend = FALSE) +
+  coord_flip() +
+  theme_void()
+```
+
+```{r fig.height = 0.5}
+library(viridis)
+fig + scale_fill_viridis(discrete = TRUE, option = "viridis")
+```
+
+
+&nbsp;
+
+... or viridis' **"Magma"** palette:  
+
+```{r fig.height = 0.5}
+fig + scale_fill_viridis(discrete = TRUE, option = "magma")
+```
+
+&nbsp;
+
+### `ggsci`
+
+The `ggsci` package provides a collection of color palettes specifically designed for scientific journals. These color palettes are designed to enhance the visual appeal and interpretability of ggplot2 graphics.  For more details, see [here](https://cran.r-project.org/web/packages/ggsci/vignettes/ggsci.html and https://nanx.me/ggsci/articles/ggsci.html).
+
+Some examples of commonly used `ggsci` palettes include:
+
+**JAMA palette:**
+
+```{r fig.height = 0.5}
+library(ggsci)
+fig + scale_fill_jama()
+```
+
+**Lancet palette:**
+
+```{r fig.height = 0.5}
+fig + scale_fill_lancet() 
+```
+
+**JCO palette:**
+
+```{r fig.height = 0.5}
+fig + scale_fill_jco()
+```
+
+&nbsp;
+
+### `RColorBrewer`
+
+ColorBrewer is a flexible tool for selecting color schemes. ColorBrewer includes color palettes that are colorblind safe, print friendly, and perceptually uniform. It offers a wide range of colors optimized for different types of data and visualization purposes. For more details, see [here](https://colorbrewer2.org/#type=sequential&scheme=BuGn&n=3).
+
+For example, this illustrates the **"PRGn" palette**: 
+
+```{r fig.height = 0.5}
+library(RColorBrewer)
+fig + scale_fill_brewer(palette = "PRGn")
+```
+
+&nbsp;
+
+You can explore all colorblind friendly ColorBrewer palettes by running: 
+
+```{r fig.height = 6.5}
+display.brewer.all(colorblindFriendly = TRUE) 
+```
+
+
+*  *  *  *
+
+# Resources {#resources}
+
+- ggplot2 vignette for theme specifications: https://ggplot2.tidyverse.org/reference/theme.html
+- a comprehensive guide on ggplot2 & color palettes: https://www.cedricscherer.com/2019/08/05/a-ggplot2-tutorial-for-beautiful-plotting-in-r/
+- ggsci color palettes for scientific journals: https://cran.r-project.org/web/packages/ggsci/vignettes/ggsci.html and https://nanx.me/ggsci/articles/ggsci.html
+- viridis color maps: https://cran.r-project.org/web/packages/viridis/vignettes/intro-to-viridis.html
+- color brewer: https://colorbrewer2.org/#type=sequential&scheme=BuGn&n=3
+
+
+<!-- remove border around images -->
+<style>
+img {
+border: 0;
+}
+</style>
```

Comment:
Hi Anne, I reviewed the knitted vignette from here: R:\GEMINI\Research Projects\Rgemini\. I did some testing with plot_theme() and tried all the GEMINI colours. The Vignette is very well documented and I liked the layout. Everything has been listed in bullet points and easy to find information. The example codes are very informative. I ran the codes and there are no errors. All the GEMINI colours not only can be referenced by colour number but also by colour names, such as "Lavender Lagoon", which I really like. All the names are beautiful and elegant.  

The only issue I had was the `library(ggthemes).` The remote install doesn't work so I used git clone and sourced the function for the testing. Besides that, everything looks great. The codes run smoothly with no errors and all the colours work perfectly.

Reply:
Hi @youx-unity, thanks so much for the review! Great to hear everything worked well for you. Regarding the `library(ggthemes)` issue, this will be solved once everything has been merged into `master` (and then you can install the newest version from there, which should prevent this issue).  

I will wait for @shijiaSMH to finalize her review on the other plotting functions. Once everything has been approved, I'll merge the pull request and close this issue.

Thanks again to both of you for taking on these reviews! :-)